app/components/GearListCard/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ *
+ * GearListCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import GrayImageOverlay from 'components/shared/GrayImageOverlay';
+
+const ConstrainedImgBySize = styled.img`
+  width: 100%;
```

Comment:
Probably better to use w-100 class

Reply:
I think keeping the width and height values in the styled component aligns with the name of it. Not sure if w-100 class is necessarily better.